### PR TITLE
add task to create zaak

### PR DESCRIPTION
**Pull request by annashamray**
Issue - https://github.com/GemeenteUtrecht/camunda-worker/issues/2

### CODE REVIEW

File: src/camunda_worker/tasks/tests/test_task_create_status.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from ..tasks import CreateStatusTask
+
+ZTC_URL = "https://openzaak.utrechtproeftuin.nl/catalogi/api/v1/"
+ZRC_URL = "https://openzaak.utrechtproeftuin.nl/zaken/api/v1/"
+STATUSTYPE = f"{ZTC_URL}statustypen/7ff0bd9d-571f-47d0-8205-77ae41c3fc0b"
+ZAAK = f"{ZRC_URL}zaken/4f8b4811-5d7e-4e9b-8201-b35f5101f891"
+STATUS = f"{ZRC_URL}statussen/b7218c76-7478-41e9-a088-54d2f914a713"
+
+
+@requests_mock.Mocker(real_http=True)
```
Comment (sergei-maertens) [inline src/camunda_worker/tasks/tests/test_task_create_status.py]: Can you add a TODO to mock whatever requires `real_http`?

File: src/camunda_worker/tasks/tests/test_task_create_zaak.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+ZTC_URL = "https://openzaak.utrechtproeftuin.nl/catalogi/api/v1/"
+ZRC_URL = "https://openzaak.utrechtproeftuin.nl/zaken/api/v1/"
+ZAAKTYPE = f"{ZTC_URL}zaaktypen/abcd"
+STATUSTYPE = f"{ZTC_URL}statustypen/7ff0bd9d-571f-47d0-8205-77ae41c3fc0b"
+ZAAK = f"{ZRC_URL}zaken/4f8b4811-5d7e-4e9b-8201-b35f5101f891"
+STATUS = f"{ZRC_URL}statussen/b7218c76-7478-41e9-a088-54d2f914a713"
+
+
+@requests_mock.Mocker(real_http=True)
```
Comment (sergei-maertens) [inline src/camunda_worker/tasks/tests/test_task_create_zaak.py]: please add the same TODO

File: src/camunda_worker/tasks/tests/test_task_create_status.py (hunk 1 above)
Comment (annashamray) [reply src/camunda_worker/tasks/tests/test_task_create_status.py]: TODO is added

File: src/camunda_worker/tasks/tests/test_task_create_zaak.py (hunk 2 above)
Comment (annashamray) [reply src/camunda_worker/tasks/tests/test_task_create_zaak.py]: TODO is added

File: src/camunda_worker/tasks/admin.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+from .models import TaskMapping
+
+
+@admin.register(TaskMapping)
+class TaskMappingAdmin(admin.ModelAdmin):
+    model = TaskMapping
```
[the remark this answers is not in the document]
Comment (annashamray) [reply src/camunda_worker/tasks/admin.py]: the line is removed

**Pull request merged**